website/docs/reference/resource-properties/freshness.md

```diff
@@ -81,6 +87,50 @@ If using a non-UTC timestamp, cast it to UTC first:
 loaded_at_field: "convert_timezone('Australia/Sydney', 'UTC', created_at_local)"
 ```
 
+<VersionBlock firstVersion="1.10">
+
+## loaded_at_query
+
+Specify custom SQL to generate the `maxLoadedAt` timestamp on the source (rather than via warehouse metadata or the `loaded_at_field` config).
+
+Examples: 
+
+```yaml
+
+sources:
+  - name: your_source
+    freshness:
+      error_after:
+        count: 2
+        period: hour 
+    loaded_at_query: "select max(ordered_at) from {{ this }}"
```

Comment:
the example you currently have, i would just use `loaded_at_field` for...

i imagine a more realistic example would be selecting from an audit table:
```
select from my_audit_tables
where table_name = {{ this.identifier }}
and status = 'completed'
```

Reply:
another realistic example you could show would be baking in some kind of delay:

```
select max(_sdc_batched_at) from (
    select * from {{ this }}
    where _sdc_batched_at > dateadd(day, -7, current_date)
    qualify count(*) over (partition by _sdc_batched_at::date) > 2000
)
```

^This example is essentially saying: "I don't want to move the freshness bookmark forward, until there are at least 2,000 rows on the current calendar date."